app/Http/Controllers/Admin/ModulesController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Contracts\Controller;
+use App\Models\Module;
+use App\Services\ModuleService;
+use Illuminate\Http\Request;
+use Laracasts\Flash\Flash;
+
+class ModulesController extends Controller
+{
+    private $moduleSvc;
+
+    public function __construct(ModuleService $moduleSvc)
+    {
+        $this->moduleSvc = $moduleSvc;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @param Request $request
+     *
+     * @return mixed
+     */
+    public function index()
+    {
+        $modules = Module::all();
+        return view('admin.modules.index', [
+            'modules' => $modules,
+        ]);
+    }
+
+    public function create()
+    {
+        return view('admin.modules.create');
+    }
+
+    public function store(Request $request)
+    {
+        $array = [
+            'file'    => $request->file('module_file'),
+            'enabled' => $request->has('enabled') ? 1 : 0,
+        ];
+
+        $store = $this->moduleSvc->createModule($array);
+
+        if ($store == true) {
```

Comment:
`===`, surprised styleci didn't catch this

Reply:
Haha, Shall I use `$store `only?

Reply:
Yeah, that's fine, since you're explicitly returning a bool from the `createModule()` method